src/app/src/main/kotlin/org/imperial/mrc/hint/models/ErrorDetail.kt

```diff
@@ -0,0 +1,13 @@
+package org.imperial.mrc.hint.models
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+
+class ErrorDetail(private val httpStatus: HttpStatus, val detail: String){
+
+    val error = "OTHER_ERROR"
```

Comment:
At the moment the schema only accepts a limited number of error codes. When we relax this and/or add more codes, we can return something more sensible/informative here